docs/content/direct/examples.md

```diff
@@ -42,45 +40,21 @@ The following steps establish an initial state used in the examples below.
 
 1. Create an inventory & mailbox space of type `vcluster` running *OCM* (Open Cluster Management)
 in KubeFlex. Note that `-p ocm` runs a post-create hook on the *vcluster* control plane
-which installs OCM on it.
+which installs OCM on it. This step includes the status add-on described [here](./architecture.md#ocm-status-add-on-agent) for more details on the add-on.
```

Comment:
The architecture document describes two status add-on thingies: the "Status Add-On Controller" and the "OCM Status Add-On Agent". I recommend that the text here refer specifically to the controller.

Reply:
ok, reworded as you suggested